apmpackage/apm/data_stream/internal_metrics/elasticsearch/ingest_pipeline/default.yml

```
@@ -9,3 +9,7 @@ processors:
       name: process_ppid
   - pipeline:
       name: client_geoip
+  - script:
+      if: ctx._metric_descriptions != null
+      source: |
+        ctx.remove("_metric_descriptions");
```

```suggestion
  - remove:
      field: _metric_descriptions
      ignore_missing: true
```

Should be a bit more efficient.